### DISCUSSION TITLE
Can al-folio be used without any third-party dependencies?

### DISCUSSION THREAD

**Discussion by chuiuiui** [alshedivat/al-folio#937] [Q&A] [answered]
Hi,
I really like the visuals of al-folio - great work!
The title really says it all - I was wondering if it's possible to completely self-host the theme and avoid any requests to third-party servers. I'm aware that you can simply keep the Google Analytics flag deactivated and deactivate MathJax, but is it also possible without too much effort to modify the theme for self-hosting of jQuery, Bootstrap, and other necessary third-party dependencies?
Thanks!

**Comment by cmoralesmx** [comment]
Technically, it is possible to do so. The effort required depends on the technical experience of the person doing such change.

**Comment by george-gca** [comment, marked as answer]
Yes it is. I changed some of this to be self-hosted in #1562 so we could use purge-css.
